vta/python/vta/testing/simulator.py

```diff
@@ -55,5 +57,15 @@ def stats():
     x = tvm.get_global_func("vta.simulator.profiler_status")()
     return json.loads(x)
 
+def tsim_init(hw_lib):
+    """Init hardware library for TSIM"""
```

Comment:
for a user-facing function, always document all arguments.

```
def tsim_init(hw_lib):
     """Description
    
     Parameters
     ------------
     hw_lib : str
          Path to hardware library
"""
```

Reply:
Fixed [here](https://github.com/dmlc/tvm/pull/3258/commits/c7b7a32b37ff3dd7b7e073c25bfdf4221796274b).